.gitignore

```diff
@@ -57,6 +57,14 @@ phpunit.xml
 jackrabbit-standalone*
 jackrabbit/*
 
+# Project level ini settings, primarily for Symfony CLI
+# https://symfony.com/doc/current/setup/symfony_server.html#overriding-php-config-options-per-project
+php.ini
+
+# Project level PHP version to run with Symfony CLI
+# https://symfony.com/doc/current/setup/symfony_server.html#selecting-a-different-php-version
+.php-version
+
```

Comment:
```suggestion
# Symfony CLI
# https://symfony.com/doc/current/setup/symfony_server.html#different-php-settings-per-project
/php.ini
/.php-version
```

Would just combine them like we do it for other section and only exclude the root files for project